client/src/styles/_layout.scss

```diff
@@ -113,3 +113,15 @@ body {
 .cms .flexbox-display {
   display: flex; // Override JQuery UI, useful to override .ui-tabs-panel
 }
+
+
+/**
+ * Workaround for tooltip placement. React bootstrap add class `left`, `right`,
+ * `top` and `bottom` instead of `tooltip-left`, `tooltip-right`, `tooltip-top`
+ * and `tooltip-bottom`.
+ */
+@each $dir in [left, right, top, bottom] {
+  .tooltip.#{$dir} {
```

Comment:
Yeah, that's fine :)
We had something similar with `Label/Label.scss` (actually, that could be cleaned up to be `LabelField/LabelField.scss` later on)
And I think it's better practice to have component specific styles handled like a component specific style, even if we don't have a javascript component for it.